app/src/main/java/com/example/inharmony/SearchPresenter.java

```diff
@@ -132,11 +221,22 @@ public void selectTrack(Track item) {
             mPlayer.play(previewUrl);
         } else if (mPlayer.isPlaying()) {
             mPlayer.pause();
+
         } else {
             mPlayer.resume();
         }
     }
 
+    @Override
+    public void selectArtist(Artist artist) {
+
+    }
+
+    @Override
+    public void selectAlbum(AlbumSimple album) {
+
+    }
```

Comment:
what logic will be implemented here?

Reply:
They're empty override functions since I based them off the original search class. Just removed them from the Search interface since they're not needed!